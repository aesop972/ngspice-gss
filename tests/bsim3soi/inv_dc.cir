 Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)

.include ./nmos4p0.mod
.include ./pmos4p0.mod
.option TEMP=27C

Vpower VD 0 1.5
Vgnd VS 0 0
Vgate Gate 0 0.0
MN0 VS Gate Out VS N1 W=10u L=0.18u
MP0 VD Gate Out VS P1 W=20u L=0.18u

.dc Vgate 0 1.5 0.05
.print dc v(out)
.END
