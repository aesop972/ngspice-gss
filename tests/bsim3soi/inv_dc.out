Note: can't find init file.

Circuit:  Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)

Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Circuit:  Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
Date: Wed Nov 30 11:36:06  2005


 Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
DC transfer characteristic  Wed Nov 30 11:36:06  2005
--------------------------------------------------------------------------------
Index   sweep           v(out)          
--------------------------------------------------------------------------------
0	0.000000e+00	1.499923e+00	
1	5.000000e-02	1.499949e+00	
2	1.000000e-01	1.499948e+00	
3	1.500000e-01	1.499826e+00	
4	2.000000e-01	1.498963e+00	
5	2.500000e-01	1.495333e+00	
6	3.000000e-01	1.487043e+00	
7	3.500000e-01	1.474054e+00	
8	4.000000e-01	1.456629e+00	
9	4.500000e-01	1.434481e+00	
10	5.000000e-01	1.406612e+00	
11	5.500000e-01	1.370999e+00	
12	6.000000e-01	1.323589e+00	
13	6.500000e-01	1.254836e+00	
14	7.000000e-01	1.132041e+00	
15	7.500000e-01	6.829672e-01	
16	8.000000e-01	1.919667e-01	
17	8.500000e-01	1.139678e-01	
18	9.000000e-01	7.553252e-02	
19	9.500000e-01	5.231469e-02	
20	1.000000e+00	3.648310e-02	
21	1.050000e+00	2.492246e-02	
22	1.100000e+00	1.632612e-02	
23	1.150000e+00	1.004629e-02	
24	1.200000e+00	5.683431e-03	
25	1.250000e+00	2.900271e-03	
26	1.300000e+00	1.323044e-03	
27	1.350000e+00	5.432044e-04	
28	1.400000e+00	2.059456e-04	
29	1.450000e+00	8.022330e-05	
30	1.500000e+00	4.614839e-05	

CPU time since last call: 0.260 seconds.

Total CPU time: 0.260 seconds.

Current dynamic memory usage = 344064,
Dynamic memory limit = -18823216.

